packages/lesswrong/server/languageModels/modGPT.ts

```diff
@@ -232,7 +232,7 @@ getCollectionHooks("Comments").updateAsync.add(async ({oldDocument, newDocument}
 })
 
 getCollectionHooks("Comments").createAsync.add(async ({document}) => {
-  if (!isEAForum || !document.postId) return
+  if (!isEAForum || !document.postId || document.deleted || document.deletedPublic) return
```

Comment:
If something is deletedPublic is it not also `deleted`?

Reply:
Oh you're right, I'll update that